src/components/AssetDialog.tsx

```diff
@@ -0,0 +1,43 @@
+import AssetListItem from 'components/AssetListItem';
+import ListItem from 'components/ListItem';
+import SkeletonListItem from 'components/SkeletonListItem';
+import type { Brand } from 'store/app';
+
+const AssetDialog = ({
+  brands,
+  handleBrandSelected,
+}: {
+  brands: Array<Brand>;
+  handleBrandSelected: (brand: Brand) => void;
+}) => {
+  if (!(brands && brands.length))
+    return (
+      <div className="flex flex-col gap-4 p-5 overflow-auto ">
+        {[...Array(4).keys()].map(i => (
+          <ListItem key={i}>
+            <SkeletonListItem />
+          </ListItem>
+        ))}
+      </div>
+    );
+
+  return (
+    <div className="flex flex-col gap-4 p-5 overflow-auto ">
```

Comment:
Since this div is the same regardless of brands, consider having the conditional for what's inside and return unconditionally. e.g.,

```js
const brandSections = brand && brands.length
  ? brands.map(brand => {
…
    } 
  : [skeletonSection];
```

Then this would consistently be,

```js
return (
  <div className="flex flex-col gap-4 p-5 overflow-auto ">
    {brandSections}
```

And make clearer in code what the semantic structure is.